libs/algorithms.py

```diff
@@ -6,13 +6,35 @@
 #
 
 
-def bubbleSort(list):
-    return list
+def bubbleSort(ls):
+    is_sorted = False
+    while not is_sorted:
+        for i in range(len(ls)):
+            next_element = None
+            try:
+                next_element = ls[i+1]
+            except IndexError:
+                break
+            if ls[i] > next_element:
+                temp_i = int(ls[i+1])
+                ls[i+1] = int(ls[i])
+                ls[i] = temp_i
+        for i in range(len(ls)):
+            next_element = None
+            try:
+                next_element = ls[i+1]
+            except IndexError:
+                break
+            if ls[i] < ls[i+1]:
+                is_sorted = True
+            else:
+                is_sorted = False
+                break
```

Comment:
the reason why i made a sort of useless if statement is because i needed the `break`